tests/test_app_bokeh.py

```diff
@@ -0,0 +1,751 @@
+"""Test the tiatoolbox visualization tool."""
```

Comment:
Would it be a good idea to move these tests to a separate folder e.g., `visualization`?

Reply:
Theres only 3 for now so its probably ok as is. If any more need to be added in the future then maybe we can consider this